op-e2e/withdrawal_helper.go

```diff
@@ -216,19 +213,13 @@ func FinalizeWithdrawal(t *testing.T, cfg SystemConfig, l1Client *ethclient.Clie
 			return err == nil, nil
 		}))
 
-		resolveClaimTx, err := proxy.ResolveClaim(opts, common.Big0, common.Big0)
-		require.Nil(t, err)
-
-		resolveClaimReceipt, err = wait.ForReceiptOK(ctx, l1Client, resolveClaimTx.Hash())
-		require.Nil(t, err, "resolve claim")
-		require.Equal(t, types.ReceiptStatusSuccessful, resolveClaimReceipt.Status)
-
-		resolveTx, err := proxy.Resolve(opts)
-		require.Nil(t, err)
+		tx, err := gameContract.ResolveClaimTx(0)
+		require.NoError(t, err, "create resolveClaim tx")
+		transactions.RequireSendTx(t, ctx, l1Client, tx, privKey)
 
-		resolveReceipt, err = wait.ForReceiptOK(ctx, l1Client, resolveTx.Hash())
```

Comment:
`resolveReceipt` remains unset and is returned by this function. It's probably why `TestWithdrawals` fails.

Reply:
Good spot. Fixed.